set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241216_010006-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-16 01:00:08.744219
Config  : ./aria.localhost/private/backups/20241216_010006-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241216_010006-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241216_070005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-16 07:00:08.115923
Config  : ./aria.localhost/private/backups/20241216_070005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241216_070005-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241216_130006-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-16 13:00:11.478514
Config  : ./aria.localhost/private/backups/20241216_130006-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241216_130006-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241216_190005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-16 19:00:08.678076
Config  : ./aria.localhost/private/backups/20241216_190005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241216_190005-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241217_010006-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-17 01:00:08.850886
Config  : ./aria.localhost/private/backups/20241217_010006-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241217_010006-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241217_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-17 07:00:07.828098
Config  : ./aria.localhost/private/backups/20241217_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241217_070004-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241217_130005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-17 13:00:08.159545
Config  : ./aria.localhost/private/backups/20241217_130005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241217_130005-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241217_190004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-17 19:00:07.728702
Config  : ./aria.localhost/private/backups/20241217_190004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241217_190004-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241218_010005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-18 01:00:09.174006
Config  : ./aria.localhost/private/backups/20241218_010005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241218_010005-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241218_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-18 07:00:07.171931
Config  : ./aria.localhost/private/backups/20241218_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241218_070004-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241218_130005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-18 13:00:08.364324
Config  : ./aria.localhost/private/backups/20241218_130005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241218_130005-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241218_190004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-18 19:00:07.682685
Config  : ./aria.localhost/private/backups/20241218_190004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241218_190004-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241219_010005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-19 01:00:07.932706
Config  : ./aria.localhost/private/backups/20241219_010005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241219_010005-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241219_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-19 07:00:07.776811
Config  : ./aria.localhost/private/backups/20241219_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241219_070004-aria_localhost-database.sql.gz         283.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241219_130006-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-19 13:00:11.282283
Config  : ./aria.localhost/private/backups/20241219_130006-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241219_130006-aria_localhost-database.sql.gz         291.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241219_190004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-19 19:00:07.986683
Config  : ./aria.localhost/private/backups/20241219_190004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241219_190004-aria_localhost-database.sql.gz         291.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241220_010005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-20 01:00:08.302971
Config  : ./aria.localhost/private/backups/20241220_010005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241220_010005-aria_localhost-database.sql.gz         291.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241220_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-20 07:00:07.145574
Config  : ./aria.localhost/private/backups/20241220_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241220_070004-aria_localhost-database.sql.gz         291.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241220_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-20 13:00:07.085991
Config  : ./aria.localhost/private/backups/20241220_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241220_130004-aria_localhost-database.sql.gz         291.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241220_190005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-20 19:00:08.037510
Config  : ./aria.localhost/private/backups/20241220_190005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241220_190005-aria_localhost-database.sql.gz         291.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241221_010004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-21 01:00:08.004354
Config  : ./aria.localhost/private/backups/20241221_010004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241221_010004-aria_localhost-database.sql.gz         291.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241221_070005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-21 07:00:12.017119
Config  : ./aria.localhost/private/backups/20241221_070005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241221_070005-aria_localhost-database.sql.gz         291.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241221_130005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-21 13:00:08.419226
Config  : ./aria.localhost/private/backups/20241221_130005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241221_130005-aria_localhost-database.sql.gz         291.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241221_190004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-21 19:00:08.283180
Config  : ./aria.localhost/private/backups/20241221_190004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241221_190004-aria_localhost-database.sql.gz         291.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241222_010005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-22 01:00:07.742548
Config  : ./aria.localhost/private/backups/20241222_010005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241222_010005-aria_localhost-database.sql.gz         291.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241222_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-22 07:00:06.861971
Config  : ./aria.localhost/private/backups/20241222_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241222_070004-aria_localhost-database.sql.gz         291.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241222_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-22 13:00:08.445755
Config  : ./aria.localhost/private/backups/20241222_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241222_130004-aria_localhost-database.sql.gz         292.4KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241222_190004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-22 19:00:12.104036
Config  : ./aria.localhost/private/backups/20241222_190004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241222_190004-aria_localhost-database.sql.gz         299.1KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241223_010006-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-23 01:00:15.502957
Config  : ./aria.localhost/private/backups/20241223_010006-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241223_010006-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241223_070005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-23 07:00:08.023637
Config  : ./aria.localhost/private/backups/20241223_070005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241223_070005-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241223_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-23 13:00:07.953536
Config  : ./aria.localhost/private/backups/20241223_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241223_130004-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241223_190005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-23 19:00:08.664940
Config  : ./aria.localhost/private/backups/20241223_190005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241223_190005-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241224_010005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-24 01:00:07.784070
Config  : ./aria.localhost/private/backups/20241224_010005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241224_010005-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241224_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-24 07:00:07.956438
Config  : ./aria.localhost/private/backups/20241224_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241224_070004-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241224_130005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-24 13:00:08.657037
Config  : ./aria.localhost/private/backups/20241224_130005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241224_130005-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241224_190006-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-24 19:00:12.042369
Config  : ./aria.localhost/private/backups/20241224_190006-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241224_190006-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241225_010005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-25 01:00:12.062810
Config  : ./aria.localhost/private/backups/20241225_010005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241225_010005-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241225_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-25 07:00:11.535279
Config  : ./aria.localhost/private/backups/20241225_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241225_070004-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241225_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-25 13:00:11.132849
Config  : ./aria.localhost/private/backups/20241225_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241225_130004-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241225_190004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-25 19:00:10.319272
Config  : ./aria.localhost/private/backups/20241225_190004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241225_190004-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241226_010005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-26 01:00:09.690234
Config  : ./aria.localhost/private/backups/20241226_010005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241226_010005-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241226_070005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-26 07:00:07.888165
Config  : ./aria.localhost/private/backups/20241226_070005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241226_070005-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241226_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-26 13:00:07.814661
Config  : ./aria.localhost/private/backups/20241226_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241226_130004-aria_localhost-database.sql.gz         302.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241226_190005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-26 19:00:10.821058
Config  : ./aria.localhost/private/backups/20241226_190005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241226_190005-aria_localhost-database.sql.gz         316.9KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241227_010005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-27 01:00:11.839474
Config  : ./aria.localhost/private/backups/20241227_010005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241227_010005-aria_localhost-database.sql.gz         344.5KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241227_070003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-27 07:00:07.441893
Config  : ./aria.localhost/private/backups/20241227_070003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241227_070003-aria_localhost-database.sql.gz         350.5KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241227_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-27 13:00:07.872009
Config  : ./aria.localhost/private/backups/20241227_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241227_130004-aria_localhost-database.sql.gz         369.8KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241227_190004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-27 19:00:07.790683
Config  : ./aria.localhost/private/backups/20241227_190004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241227_190004-aria_localhost-database.sql.gz         376.2KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241228_010004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-28 01:00:07.742246
Config  : ./aria.localhost/private/backups/20241228_010004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241228_010004-aria_localhost-database.sql.gz         382.8KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241228_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-28 07:00:07.707919
Config  : ./aria.localhost/private/backups/20241228_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241228_070004-aria_localhost-database.sql.gz         388.8KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241228_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-28 13:00:07.350443
Config  : ./aria.localhost/private/backups/20241228_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241228_130004-aria_localhost-database.sql.gz         395.0KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241228_190003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-28 19:00:13.562441
Config  : ./aria.localhost/private/backups/20241228_190003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241228_190003-aria_localhost-database.sql.gz         401.1KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241229_010004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-29 01:00:08.105052
Config  : ./aria.localhost/private/backups/20241229_010004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241229_010004-aria_localhost-database.sql.gz         408.2KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241229_070003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-29 07:00:07.924320
Config  : ./aria.localhost/private/backups/20241229_070003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241229_070003-aria_localhost-database.sql.gz         414.4KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241229_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-29 13:00:07.965357
Config  : ./aria.localhost/private/backups/20241229_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241229_130004-aria_localhost-database.sql.gz         423.1KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241229_190004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-29 19:00:09.600057
Config  : ./aria.localhost/private/backups/20241229_190004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241229_190004-aria_localhost-database.sql.gz         429.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241230_010004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-30 01:00:08.684762
Config  : ./aria.localhost/private/backups/20241230_010004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241230_010004-aria_localhost-database.sql.gz         423.1KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241230_070003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-30 07:00:07.228459
Config  : ./aria.localhost/private/backups/20241230_070003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241230_070003-aria_localhost-database.sql.gz         429.5KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241230_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-30 13:00:10.859680
Config  : ./aria.localhost/private/backups/20241230_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241230_130004-aria_localhost-database.sql.gz         435.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241230_190004-aria_localhost-database.sql.gz

b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
Backup failed for Site aria.localhost. Database or site_config.json may be corrupted
Traceback with variables (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 848, in backup
    odb = scheduled_backup(
      context = {'sites': ['aria.localhost'], 'force': False, 'verbose': True, 'profile': False}
      with_files = False
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_backup_conf = False
      verbose = True
      compress = False
      include = ''
      exclude = ''
      old_backup_metadata = False
      scheduled_backup = <function scheduled_backup at 0xffffb578a4d0>
      exit_code = 0
      rollback_callback = None
      site = 'aria.localhost'
  File "apps/frappe/frappe/utils/backups.py", line 568, in scheduled_backup
    return new_backup(
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffffb47a3ee0>
  File "apps/frappe/frappe/utils/backups.py", line 626, in new_backup
    odb.get_backup(older_than, ignore_files, force=force)
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffffb47a3ee0>
      odb = <frappe.utils.backups.BackupGenerator object at 0xffffb47a3f70>
  File "apps/frappe/frappe/utils/backups.py", line 195, in get_backup
    self.delete_if_step_fails(self.take_dump, self.backup_path_db)
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb47a3f70>
      older_than = 6
      ignore_files = True
      force = True
      last_db = False
      last_file = False
      last_private_file = False
      site_config_backup_path = False
  File "apps/frappe/frappe/utils/backups.py", line 520, in delete_if_step_fails
    raise e
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb47a3f70>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffffb47a3f70>>
      paths = ('./aria.localhost/private/backups/20241230_190004-aria_localhost-database.sql.gz',)
      path = './aria.localhost/private/backups/20241230_190004-aria_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 515, in delete_if_step_fails
    step()
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb47a3f70>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffffb47a3f70>>
      paths = ('./aria.localhost/private/backups/20241230_190004-aria_localhost-database.sql.gz',)
      path = './aria.localhost/private/backups/20241230_190004-aria_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 468, in take_dump
    frappe.utils.execute_in_shell(command, low_priority=True, check_exit_code=True)
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      frappe = <module 'frappe' from 'apps/frappe/frappe/__init__.py'>
      get_app_branch = <function get_app_branch at 0xffffb5febe20>
      gzip_exc = '/usr/bin/gzip'
      database_header_content = ['begin frappe metadata', '[frappe]', 'version = 15.50.0', 'branch = version-15', 'end frappe metadata', '']
      generated_header = '-- begin frappe metadata\n-- [frappe]\n-- version = 15.50.0\n-- branch = version-15\n-- end frappe metadata\n-- \n'
      f = <_io.TextIOWrapper name='./aria.localhost/private/backups/20241230_190004-aria_localhost-database.sql.gz' encoding='UTF-8'>
      cmd = ['/usr/bin/mariadb-dump', '--user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee']
      extra = []
      get_command = <function get_command at 0xffffb4a27a30>
      bin = '/usr/bin/mariadb-dump'
      args = ['--user=_8fa9d71ae1ff9bee', '--host=127.0.0.1', '--port=3306', '--password=9XVbuEAXBcFsevpg', '--single-transaction', '--quick', '--lock-tables=false', '_8fa9d71ae1ff9bee']
      bin_name = 'mariadb-dump'
      command = 'set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241230_190004-aria_localhost-database.sql.gz'
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb47a3f70>
  File "apps/frappe/frappe/utils/__init__.py", line 484, in execute_in_shell
    raise frappe.CommandFailedError(
      cmd = 'set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241230_190004-aria_localhost-database.sql.gz'
      verbose = False
      low_priority = True
      check_exit_code = True
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      tempfile = <module 'tempfile' from '/usr/lib/python3.10/tempfile.py'>
      Popen = <class 'subprocess.Popen'>
      stdout = <_io.BufferedRandom name=6>
      stderr = <_io.BufferedRandom name=7>
      kwargs = <exception while printing> Traceback (most recent call last):
          File "env/lib/python3.10/site-packages/traceback_with_variables/core.py", line 222, in _to_cropped_str
            raw = print_(obj)
          File "apps/frappe/frappe/utils/__init__.py", line 326, in dict_printer
            v = deepcopy(v)
          File "/usr/lib/python3.10/copy.py", line 146, in deepcopy
            y = copier(x, memo)
          File "/usr/lib/python3.10/copy.py", line 231, in _deepcopy_dict
            y[deepcopy(key, memo)] = deepcopy(value, memo)
          File "/usr/lib/python3.10/copy.py", line 161, in deepcopy
            rv = reductor(4)
        TypeError: cannot pickle '_io.BufferedRandom' object
        
      p = <Popen: returncode: 2 args: 'set -o pipefail; /usr/bin/mariadb-dump --user=_...>
      exit_code = 2
      out = b''
      err = b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
      failed = 2
frappe.exceptions.CommandFailedError: Command failed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241231_010004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-31 01:00:08.516609
Config  : ./aria.localhost/private/backups/20241231_010004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241231_010004-aria_localhost-database.sql.gz         443.5KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241231_070003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-31 07:00:07.964889
Config  : ./aria.localhost/private/backups/20241231_070003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241231_070003-aria_localhost-database.sql.gz         450.0KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241231_130003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2024-12-31 13:00:07.065528
Config  : ./aria.localhost/private/backups/20241231_130003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20241231_130003-aria_localhost-database.sql.gz         456.1KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241231_190005-aria_localhost-database.sql.gz

Backup failed for Site aria.localhost. Database or site_config.json may be corrupted
Traceback with variables (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 848, in backup
    odb = scheduled_backup(
      context = {'sites': ['aria.localhost'], 'force': False, 'verbose': True, 'profile': False}
      with_files = False
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_backup_conf = False
      verbose = True
      compress = False
      include = ''
      exclude = ''
      old_backup_metadata = False
      scheduled_backup = <function scheduled_backup at 0xffff9445a4d0>
      exit_code = 0
      rollback_callback = None
      site = 'aria.localhost'
  File "apps/frappe/frappe/utils/backups.py", line 568, in scheduled_backup
    return new_backup(
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffff93473ee0>
  File "apps/frappe/frappe/utils/backups.py", line 626, in new_backup
    odb.get_backup(older_than, ignore_files, force=force)
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffff93473ee0>
      odb = <frappe.utils.backups.BackupGenerator object at 0xffff93473f70>
  File "apps/frappe/frappe/utils/backups.py", line 195, in get_backup
    self.delete_if_step_fails(self.take_dump, self.backup_path_db)
      self = <frappe.utils.backups.BackupGenerator object at 0xffff93473f70>
      older_than = 6
      ignore_files = True
      force = True
      last_db = False
      last_file = False
      last_private_file = False
      site_config_backup_path = False
  File "apps/frappe/frappe/utils/backups.py", line 520, in delete_if_step_fails
    raise e
      self = <frappe.utils.backups.BackupGenerator object at 0xffff93473f70>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffff93473f70>>
      paths = ('./aria.localhost/private/backups/20241231_190005-aria_localhost-database.sql.gz',)
      path = './aria.localhost/private/backups/20241231_190005-aria_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 515, in delete_if_step_fails
    step()
      self = <frappe.utils.backups.BackupGenerator object at 0xffff93473f70>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffff93473f70>>
      paths = ('./aria.localhost/private/backups/20241231_190005-aria_localhost-database.sql.gz',)
      path = './aria.localhost/private/backups/20241231_190005-aria_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 468, in take_dump
    frappe.utils.execute_in_shell(command, low_priority=True, check_exit_code=True)
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      frappe = <module 'frappe' from 'apps/frappe/frappe/__init__.py'>
      get_app_branch = <function get_app_branch at 0xffff94cbbe20>
      gzip_exc = '/usr/bin/gzip'
      database_header_content = ['begin frappe metadata', '[frappe]', 'version = 15.50.0', 'branch = version-15', 'end frappe metadata', '']
      generated_header = '-- begin frappe metadata\n-- [frappe]\n-- version = 15.50.0\n-- branch = version-15\n-- end frappe metadata\n-- \n'
      f = <_io.TextIOWrapper name='./aria.localhost/private/backups/20241231_190005-aria_localhost-database.sql.gz' encoding='UTF-8'>
      cmd = ['/usr/bin/mariadb-dump', '--user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee']
      extra = []
      get_command = <function get_command at 0xffff936f7a30>
      bin = '/usr/bin/mariadb-dump'
      args = ['--user=_8fa9d71ae1ff9bee', '--host=127.0.0.1', '--port=3306', '--password=9XVbuEAXBcFsevpg', '--single-transaction', '--quick', '--lock-tables=false', '_8fa9d71ae1ff9bee']
      bin_name = 'mariadb-dump'
      command = 'set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241231_190005-aria_localhost-database.sql.gz'
      self = <frappe.utils.backups.BackupGenerator object at 0xffff93473f70>
  File "apps/frappe/frappe/utils/__init__.py", line 484, in execute_in_shell
    raise frappe.CommandFailedError(
      cmd = 'set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20241231_190005-aria_localhost-database.sql.gz'
      verbose = False
      low_priority = True
      check_exit_code = True
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      tempfile = <module 'tempfile' from '/usr/lib/python3.10/tempfile.py'>
      Popen = <class 'subprocess.Popen'>
      stdout = <_io.BufferedRandom name=6>
      stderr = <_io.BufferedRandom name=7>
      kwargs = <exception while printing> Traceback (most recent call last):
          File "env/lib/python3.10/site-packages/traceback_with_variables/core.py", line 222, in _to_cropped_str
            raw = print_(obj)
          File "apps/frappe/frappe/utils/__init__.py", line 326, in dict_printer
            v = deepcopy(v)
          File "/usr/lib/python3.10/copy.py", line 146, in deepcopy
            y = copier(x, memo)
          File "/usr/lib/python3.10/copy.py", line 231, in _deepcopy_dict
            y[deepcopy(key, memo)] = deepcopy(value, memo)
          File "/usr/lib/python3.10/copy.py", line 161, in deepcopy
            rv = reductor(4)
        TypeError: cannot pickle '_io.BufferedRandom' object
        
      p = <Popen: returncode: 2 args: 'set -o pipefail; /usr/bin/mariadb-dump --user=_...>
      exit_code = 2
      out = b''
      err = b''
      failed = 2
frappe.exceptions.CommandFailedError: Command failed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250101_010004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-01 01:00:08.820135
Config  : ./aria.localhost/private/backups/20250101_010004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250101_010004-aria_localhost-database.sql.gz         468.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250101_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-01 07:00:07.340294
Config  : ./aria.localhost/private/backups/20250101_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250101_070004-aria_localhost-database.sql.gz         474.8KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250101_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-01 13:00:07.956591
Config  : ./aria.localhost/private/backups/20250101_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250101_130004-aria_localhost-database.sql.gz         480.9KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250101_190004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-01 19:00:07.789446
Config  : ./aria.localhost/private/backups/20250101_190004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250101_190004-aria_localhost-database.sql.gz         487.2KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250102_010004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-02 01:00:12.621212
Config  : ./aria.localhost/private/backups/20250102_010004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250102_010004-aria_localhost-database.sql.gz         494.0KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250102_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-02 07:00:07.498501
Config  : ./aria.localhost/private/backups/20250102_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250102_070004-aria_localhost-database.sql.gz         500.1KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250102_130003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-02 13:00:07.631225
Config  : ./aria.localhost/private/backups/20250102_130003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250102_130003-aria_localhost-database.sql.gz         506.2KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250102_190003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-02 19:00:08.015702
Config  : ./aria.localhost/private/backups/20250102_190003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250102_190003-aria_localhost-database.sql.gz         512.2KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250103_010004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-03 01:00:07.944341
Config  : ./aria.localhost/private/backups/20250103_010004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250103_010004-aria_localhost-database.sql.gz         509.0KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250103_070003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-03 07:00:07.117531
Config  : ./aria.localhost/private/backups/20250103_070003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250103_070003-aria_localhost-database.sql.gz         515.1KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250103_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-03 13:00:08.326274
Config  : ./aria.localhost/private/backups/20250103_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250103_130004-aria_localhost-database.sql.gz         521.2KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250103_190003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-03 19:00:07.803827
Config  : ./aria.localhost/private/backups/20250103_190003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250103_190003-aria_localhost-database.sql.gz         527.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250104_010004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-04 01:00:10.141611
Config  : ./aria.localhost/private/backups/20250104_010004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250104_010004-aria_localhost-database.sql.gz         509.2KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250104_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-04 07:00:08.190233
Config  : ./aria.localhost/private/backups/20250104_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250104_070004-aria_localhost-database.sql.gz         515.2KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250104_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-04 13:00:09.256526
Config  : ./aria.localhost/private/backups/20250104_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250104_130004-aria_localhost-database.sql.gz         521.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250104_190004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-04 19:00:08.979587
Config  : ./aria.localhost/private/backups/20250104_190004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250104_190004-aria_localhost-database.sql.gz         527.1KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250105_010005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-05 01:00:12.599697
Config  : ./aria.localhost/private/backups/20250105_010005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250105_010005-aria_localhost-database.sql.gz         509.1KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250105_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-05 07:00:09.682941
Config  : ./aria.localhost/private/backups/20250105_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250105_070004-aria_localhost-database.sql.gz         515.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250105_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-05 13:00:09.007734
Config  : ./aria.localhost/private/backups/20250105_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250105_130004-aria_localhost-database.sql.gz         521.4KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250105_190010-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-05 19:00:18.113166
Config  : ./aria.localhost/private/backups/20250105_190010-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250105_190010-aria_localhost-database.sql.gz         527.5KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250106_010005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-06 01:00:09.083872
Config  : ./aria.localhost/private/backups/20250106_010005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250106_010005-aria_localhost-database.sql.gz         508.5KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250106_070003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-06 07:00:07.642612
Config  : ./aria.localhost/private/backups/20250106_070003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250106_070003-aria_localhost-database.sql.gz         514.5KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250106_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-06 13:00:07.905938
Config  : ./aria.localhost/private/backups/20250106_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250106_130004-aria_localhost-database.sql.gz         520.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250106_190004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-06 19:00:09.010772
Config  : ./aria.localhost/private/backups/20250106_190004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250106_190004-aria_localhost-database.sql.gz         526.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250107_010010-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-07 01:00:22.356971
Config  : ./aria.localhost/private/backups/20250107_010010-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250107_010010-aria_localhost-database.sql.gz         513.2KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250107_070004-aria_localhost-database.sql.gz

b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
Backup failed for Site aria.localhost. Database or site_config.json may be corrupted
Traceback with variables (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 848, in backup
    odb = scheduled_backup(
      context = {'sites': ['aria.localhost'], 'force': False, 'verbose': True, 'profile': False}
      with_files = False
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_backup_conf = False
      verbose = True
      compress = False
      include = ''
      exclude = ''
      old_backup_metadata = False
      scheduled_backup = <function scheduled_backup at 0xffff8eada4d0>
      exit_code = 0
      rollback_callback = None
      site = 'aria.localhost'
  File "apps/frappe/frappe/utils/backups.py", line 568, in scheduled_backup
    return new_backup(
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffff8daf3ee0>
  File "apps/frappe/frappe/utils/backups.py", line 626, in new_backup
    odb.get_backup(older_than, ignore_files, force=force)
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffff8daf3ee0>
      odb = <frappe.utils.backups.BackupGenerator object at 0xffff8daf3f70>
  File "apps/frappe/frappe/utils/backups.py", line 195, in get_backup
    self.delete_if_step_fails(self.take_dump, self.backup_path_db)
      self = <frappe.utils.backups.BackupGenerator object at 0xffff8daf3f70>
      older_than = 6
      ignore_files = True
      force = True
      last_db = False
      last_file = False
      last_private_file = False
      site_config_backup_path = False
  File "apps/frappe/frappe/utils/backups.py", line 520, in delete_if_step_fails
    raise e
      self = <frappe.utils.backups.BackupGenerator object at 0xffff8daf3f70>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffff8daf3f70>>
      paths = ('./aria.localhost/private/backups/20250107_070004-aria_localhost-database.sql.gz',)
      path = './aria.localhost/private/backups/20250107_070004-aria_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 515, in delete_if_step_fails
    step()
      self = <frappe.utils.backups.BackupGenerator object at 0xffff8daf3f70>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffff8daf3f70>>
      paths = ('./aria.localhost/private/backups/20250107_070004-aria_localhost-database.sql.gz',)
      path = './aria.localhost/private/backups/20250107_070004-aria_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 468, in take_dump
    frappe.utils.execute_in_shell(command, low_priority=True, check_exit_code=True)
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      frappe = <module 'frappe' from 'apps/frappe/frappe/__init__.py'>
      get_app_branch = <function get_app_branch at 0xffff8f33be20>
      gzip_exc = '/usr/bin/gzip'
      database_header_content = ['begin frappe metadata', '[frappe]', 'version = 15.50.0', 'branch = version-15', 'end frappe metadata', '']
      generated_header = '-- begin frappe metadata\n-- [frappe]\n-- version = 15.50.0\n-- branch = version-15\n-- end frappe metadata\n-- \n'
      f = <_io.TextIOWrapper name='./aria.localhost/private/backups/20250107_070004-aria_localhost-database.sql.gz' encoding='UTF-8'>
      cmd = ['/usr/bin/mariadb-dump', '--user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee']
      extra = []
      get_command = <function get_command at 0xffff8dd77a30>
      bin = '/usr/bin/mariadb-dump'
      args = ['--user=_8fa9d71ae1ff9bee', '--host=127.0.0.1', '--port=3306', '--password=9XVbuEAXBcFsevpg', '--single-transaction', '--quick', '--lock-tables=false', '_8fa9d71ae1ff9bee']
      bin_name = 'mariadb-dump'
      command = 'set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250107_070004-aria_localhost-database.sql.gz'
      self = <frappe.utils.backups.BackupGenerator object at 0xffff8daf3f70>
  File "apps/frappe/frappe/utils/__init__.py", line 484, in execute_in_shell
    raise frappe.CommandFailedError(
      cmd = 'set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250107_070004-aria_localhost-database.sql.gz'
      verbose = False
      low_priority = True
      check_exit_code = True
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      tempfile = <module 'tempfile' from '/usr/lib/python3.10/tempfile.py'>
      Popen = <class 'subprocess.Popen'>
      stdout = <_io.BufferedRandom name=6>
      stderr = <_io.BufferedRandom name=7>
      kwargs = <exception while printing> Traceback (most recent call last):
          File "env/lib/python3.10/site-packages/traceback_with_variables/core.py", line 222, in _to_cropped_str
            raw = print_(obj)
          File "apps/frappe/frappe/utils/__init__.py", line 326, in dict_printer
            v = deepcopy(v)
          File "/usr/lib/python3.10/copy.py", line 146, in deepcopy
            y = copier(x, memo)
          File "/usr/lib/python3.10/copy.py", line 231, in _deepcopy_dict
            y[deepcopy(key, memo)] = deepcopy(value, memo)
          File "/usr/lib/python3.10/copy.py", line 161, in deepcopy
            rv = reductor(4)
        TypeError: cannot pickle '_io.BufferedRandom' object
        
      p = <Popen: returncode: 2 args: 'set -o pipefail; /usr/bin/mariadb-dump --user=_...>
      exit_code = 2
      out = b''
      err = b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
      failed = 2
frappe.exceptions.CommandFailedError: Command failed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250107_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-07 13:00:09.107270
Config  : ./aria.localhost/private/backups/20250107_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250107_130004-aria_localhost-database.sql.gz         515.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250107_190005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-07 19:00:16.912647
Config  : ./aria.localhost/private/backups/20250107_190005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250107_190005-aria_localhost-database.sql.gz         521.5KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250108_010004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-08 01:00:07.756109
Config  : ./aria.localhost/private/backups/20250108_010004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250108_010004-aria_localhost-database.sql.gz         503.8KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250108_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-08 07:00:08.440531
Config  : ./aria.localhost/private/backups/20250108_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250108_070004-aria_localhost-database.sql.gz         510.0KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250108_130009-aria_localhost-database.sql.gz

b'mariadb-dump: Error: \'Lost connection to server during query\' when trying to dump tablespaces\nmariadb-dump: Got error: 2006: "Server has gone away" when selecting the database\n'
Backup failed for Site aria.localhost. Database or site_config.json may be corrupted
Traceback with variables (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 848, in backup
    odb = scheduled_backup(
      context = {'sites': ['aria.localhost'], 'force': False, 'verbose': True, 'profile': False}
      with_files = False
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_backup_conf = False
      verbose = True
      compress = False
      include = ''
      exclude = ''
      old_backup_metadata = False
      scheduled_backup = <function scheduled_backup at 0xffff9f7924d0>
      exit_code = 0
      rollback_callback = None
      site = 'aria.localhost'
  File "apps/frappe/frappe/utils/backups.py", line 568, in scheduled_backup
    return new_backup(
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffff9e804070>
  File "apps/frappe/frappe/utils/backups.py", line 626, in new_backup
    odb.get_backup(older_than, ignore_files, force=force)
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffff9e804070>
      odb = <frappe.utils.backups.BackupGenerator object at 0xffff9e804100>
  File "apps/frappe/frappe/utils/backups.py", line 195, in get_backup
    self.delete_if_step_fails(self.take_dump, self.backup_path_db)
      self = <frappe.utils.backups.BackupGenerator object at 0xffff9e804100>
      older_than = 6
      ignore_files = True
      force = True
      last_db = False
      last_file = False
      last_private_file = False
      site_config_backup_path = False
  File "apps/frappe/frappe/utils/backups.py", line 520, in delete_if_step_fails
    raise e
      self = <frappe.utils.backups.BackupGenerator object at 0xffff9e804100>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffff9e804100>>
      paths = ('./aria.localhost/private/backups/20250108_130009-aria_localhost-database.sql.gz',)
      path = './aria.localhost/private/backups/20250108_130009-aria_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 515, in delete_if_step_fails
    step()
      self = <frappe.utils.backups.BackupGenerator object at 0xffff9e804100>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffff9e804100>>
      paths = ('./aria.localhost/private/backups/20250108_130009-aria_localhost-database.sql.gz',)
      path = './aria.localhost/private/backups/20250108_130009-aria_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 468, in take_dump
    frappe.utils.execute_in_shell(command, low_priority=True, check_exit_code=True)
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      frappe = <module 'frappe' from 'apps/frappe/frappe/__init__.py'>
      get_app_branch = <function get_app_branch at 0xffff9ffebe20>
      gzip_exc = '/usr/bin/gzip'
      database_header_content = ['begin frappe metadata', '[frappe]', 'version = 15.50.0', 'branch = version-15', 'end frappe metadata', '']
      generated_header = '-- begin frappe metadata\n-- [frappe]\n-- version = 15.50.0\n-- branch = version-15\n-- end frappe metadata\n-- \n'
      f = <_io.TextIOWrapper name='./aria.localhost/private/backups/20250108_130009-aria_localhost-database.sql.gz' encoding='UTF-8'>
      cmd = ['/usr/bin/mariadb-dump', '--user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee']
      extra = []
      get_command = <function get_command at 0xffff9ea33a30>
      bin = '/usr/bin/mariadb-dump'
      args = ['--user=_8fa9d71ae1ff9bee', '--host=127.0.0.1', '--port=3306', '--password=9XVbuEAXBcFsevpg', '--single-transaction', '--quick', '--lock-tables=false', '_8fa9d71ae1ff9bee']
      bin_name = 'mariadb-dump'
      command = 'set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250108_130009-aria_localhost-database.sql.gz'
      self = <frappe.utils.backups.BackupGenerator object at 0xffff9e804100>
  File "apps/frappe/frappe/utils/__init__.py", line 484, in execute_in_shell
    raise frappe.CommandFailedError(
      cmd = 'set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250108_130009-aria_localhost-database.sql.gz'
      verbose = False
      low_priority = True
      check_exit_code = True
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      tempfile = <module 'tempfile' from '/usr/lib/python3.10/tempfile.py'>
      Popen = <class 'subprocess.Popen'>
      stdout = <_io.BufferedRandom name=6>
      stderr = <_io.BufferedRandom name=7>
      kwargs = <exception while printing> Traceback (most recent call last):
          File "env/lib/python3.10/site-packages/traceback_with_variables/core.py", line 222, in _to_cropped_str
            raw = print_(obj)
          File "apps/frappe/frappe/utils/__init__.py", line 326, in dict_printer
            v = deepcopy(v)
          File "/usr/lib/python3.10/copy.py", line 146, in deepcopy
            y = copier(x, memo)
          File "/usr/lib/python3.10/copy.py", line 231, in _deepcopy_dict
            y[deepcopy(key, memo)] = deepcopy(value, memo)
          File "/usr/lib/python3.10/copy.py", line 161, in deepcopy
            rv = reductor(4)
        TypeError: cannot pickle '_io.BufferedRandom' object
        
      p = <Popen: returncode: 2 args: 'set -o pipefail; /usr/bin/mariadb-dump --user=_...>
      exit_code = 2
      out = b''
      err = b'mariadb-dump: Error: \'Lost connection to server during query\' when trying to dump tablespaces\nmariadb-dump: Got error: 2006: "Server has gone away" when selecting the database\n'
      failed = 2
frappe.exceptions.CommandFailedError: Command failed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250108_190005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-08 19:00:11.228483
Config  : ./aria.localhost/private/backups/20250108_190005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250108_190005-aria_localhost-database.sql.gz         616.8KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250109_010005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-09 01:00:10.578954
Config  : ./aria.localhost/private/backups/20250109_010005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250109_010005-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250109_070004-aria_localhost-database.sql.gz

b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
Backup failed for Site aria.localhost. Database or site_config.json may be corrupted
Traceback with variables (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 848, in backup
    odb = scheduled_backup(
      context = {'sites': ['aria.localhost'], 'force': False, 'verbose': True, 'profile': False}
      with_files = False
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_backup_conf = False
      verbose = True
      compress = False
      include = ''
      exclude = ''
      old_backup_metadata = False
      scheduled_backup = <function scheduled_backup at 0xffffb76b24d0>
      exit_code = 0
      rollback_callback = None
      site = 'aria.localhost'
  File "apps/frappe/frappe/utils/backups.py", line 568, in scheduled_backup
    return new_backup(
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffffb6724070>
  File "apps/frappe/frappe/utils/backups.py", line 626, in new_backup
    odb.get_backup(older_than, ignore_files, force=force)
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffffb6724070>
      odb = <frappe.utils.backups.BackupGenerator object at 0xffffb6724100>
  File "apps/frappe/frappe/utils/backups.py", line 195, in get_backup
    self.delete_if_step_fails(self.take_dump, self.backup_path_db)
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb6724100>
      older_than = 6
      ignore_files = True
      force = True
      last_db = False
      last_file = False
      last_private_file = False
      site_config_backup_path = False
  File "apps/frappe/frappe/utils/backups.py", line 520, in delete_if_step_fails
    raise e
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb6724100>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffffb6724100>>
      paths = ('./aria.localhost/private/backups/20250109_070004-aria_localhost-database.sql.gz',)
      path = './aria.localhost/private/backups/20250109_070004-aria_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 515, in delete_if_step_fails
    step()
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb6724100>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffffb6724100>>
      paths = ('./aria.localhost/private/backups/20250109_070004-aria_localhost-database.sql.gz',)
      path = './aria.localhost/private/backups/20250109_070004-aria_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 468, in take_dump
    frappe.utils.execute_in_shell(command, low_priority=True, check_exit_code=True)
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      frappe = <module 'frappe' from 'apps/frappe/frappe/__init__.py'>
      get_app_branch = <function get_app_branch at 0xffffb7f0be20>
      gzip_exc = '/usr/bin/gzip'
      database_header_content = ['begin frappe metadata', '[frappe]', 'version = 15.50.0', 'branch = version-15', 'end frappe metadata', '']
      generated_header = '-- begin frappe metadata\n-- [frappe]\n-- version = 15.50.0\n-- branch = version-15\n-- end frappe metadata\n-- \n'
      f = <_io.TextIOWrapper name='./aria.localhost/private/backups/20250109_070004-aria_localhost-database.sql.gz' encoding='UTF-8'>
      cmd = ['/usr/bin/mariadb-dump', '--user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee']
      extra = []
      get_command = <function get_command at 0xffffb6953a30>
      bin = '/usr/bin/mariadb-dump'
      args = ['--user=_8fa9d71ae1ff9bee', '--host=127.0.0.1', '--port=3306', '--password=9XVbuEAXBcFsevpg', '--single-transaction', '--quick', '--lock-tables=false', '_8fa9d71ae1ff9bee']
      bin_name = 'mariadb-dump'
      command = 'set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250109_070004-aria_localhost-database.sql.gz'
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb6724100>
  File "apps/frappe/frappe/utils/__init__.py", line 484, in execute_in_shell
    raise frappe.CommandFailedError(
      cmd = 'set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=9XVbuEAXBcFsevpg --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250109_070004-aria_localhost-database.sql.gz'
      verbose = False
      low_priority = True
      check_exit_code = True
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      tempfile = <module 'tempfile' from '/usr/lib/python3.10/tempfile.py'>
      Popen = <class 'subprocess.Popen'>
      stdout = <_io.BufferedRandom name=6>
      stderr = <_io.BufferedRandom name=7>
      kwargs = <exception while printing> Traceback (most recent call last):
          File "env/lib/python3.10/site-packages/traceback_with_variables/core.py", line 222, in _to_cropped_str
            raw = print_(obj)
          File "apps/frappe/frappe/utils/__init__.py", line 326, in dict_printer
            v = deepcopy(v)
          File "/usr/lib/python3.10/copy.py", line 146, in deepcopy
            y = copier(x, memo)
          File "/usr/lib/python3.10/copy.py", line 231, in _deepcopy_dict
            y[deepcopy(key, memo)] = deepcopy(value, memo)
          File "/usr/lib/python3.10/copy.py", line 161, in deepcopy
            rv = reductor(4)
        TypeError: cannot pickle '_io.BufferedRandom' object
        
      p = <Popen: returncode: 2 args: 'set -o pipefail; /usr/bin/mariadb-dump --user=_...>
      exit_code = 2
      out = b''
      err = b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
      failed = 2
frappe.exceptions.CommandFailedError: Command failed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250109_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-09 13:00:08.437184
Config  : ./aria.localhost/private/backups/20250109_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250109_130004-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250109_190005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-09 19:00:09.206748
Config  : ./aria.localhost/private/backups/20250109_190005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250109_190005-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250110_010007-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-10 01:00:11.499813
Config  : ./aria.localhost/private/backups/20250110_010007-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250110_010007-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250110_070005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-10 07:00:09.289860
Config  : ./aria.localhost/private/backups/20250110_070005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250110_070005-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250110_130010-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-10 13:00:17.575715
Config  : ./aria.localhost/private/backups/20250110_130010-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250110_130010-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250110_190005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-10 19:00:10.119548
Config  : ./aria.localhost/private/backups/20250110_190005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250110_190005-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250111_010006-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-11 01:00:14.036250
Config  : ./aria.localhost/private/backups/20250111_010006-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250111_010006-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250111_070005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-11 07:00:09.119366
Config  : ./aria.localhost/private/backups/20250111_070005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250111_070005-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250111_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-11 13:00:08.680787
Config  : ./aria.localhost/private/backups/20250111_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250111_130004-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250111_190005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-11 19:00:09.182833
Config  : ./aria.localhost/private/backups/20250111_190005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250111_190005-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250112_010006-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-12 01:00:10.363456
Config  : ./aria.localhost/private/backups/20250112_010006-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250112_010006-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250112_070005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-12 07:00:08.396919
Config  : ./aria.localhost/private/backups/20250112_070005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250112_070005-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250112_130005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-12 13:00:09.167357
Config  : ./aria.localhost/private/backups/20250112_130005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250112_130005-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250112_190004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-12 19:00:08.730780
Config  : ./aria.localhost/private/backups/20250112_190004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250112_190004-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250113_010006-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-13 01:00:10.085305
Config  : ./aria.localhost/private/backups/20250113_010006-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250113_010006-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250113_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-13 07:00:08.159356
Config  : ./aria.localhost/private/backups/20250113_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250113_070004-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250113_130006-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-13 13:00:10.946644
Config  : ./aria.localhost/private/backups/20250113_130006-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250113_130006-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250113_190005-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-13 19:00:09.449931
Config  : ./aria.localhost/private/backups/20250113_190005-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250113_190005-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250114_010006-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-14 01:00:09.591480
Config  : ./aria.localhost/private/backups/20250114_010006-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250114_010006-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250114_070004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-14 07:00:08.586756
Config  : ./aria.localhost/private/backups/20250114_070004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250114_070004-aria_localhost-database.sql.gz         658.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250114_130012-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-14 13:01:09.266171
Config  : ./aria.localhost/private/backups/20250114_130012-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250114_130012-aria_localhost-database.sql.gz         521.2KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250114_190003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-14 19:00:06.760575
Config  : ./aria.localhost/private/backups/20250114_190003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250114_190003-aria_localhost-database.sql.gz         569.1KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250115_010004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-15 01:00:08.719662
Config  : ./aria.localhost/private/backups/20250115_010004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250115_010004-aria_localhost-database.sql.gz         562.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250115_070003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-15 07:00:07.012940
Config  : ./aria.localhost/private/backups/20250115_070003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250115_070003-aria_localhost-database.sql.gz         568.8KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250115_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-15 13:00:08.159725
Config  : ./aria.localhost/private/backups/20250115_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250115_130004-aria_localhost-database.sql.gz         575.0KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250115_190004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-15 19:00:08.702861
Config  : ./aria.localhost/private/backups/20250115_190004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250115_190004-aria_localhost-database.sql.gz         581.1KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250116_010004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-16 01:00:07.550243
Config  : ./aria.localhost/private/backups/20250116_010004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250116_010004-aria_localhost-database.sql.gz         574.3KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250116_070003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-16 07:00:07.549772
Config  : ./aria.localhost/private/backups/20250116_070003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250116_070003-aria_localhost-database.sql.gz         580.5KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250116_130004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-16 13:00:08.587835
Config  : ./aria.localhost/private/backups/20250116_130004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250116_130004-aria_localhost-database.sql.gz         586.6KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250116_190003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-16 19:00:08.463129
Config  : ./aria.localhost/private/backups/20250116_190003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250116_190003-aria_localhost-database.sql.gz         592.8KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250117_010004-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-17 01:00:07.519130
Config  : ./aria.localhost/private/backups/20250117_010004-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250117_010004-aria_localhost-database.sql.gz         599.7KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250117_070003-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-17 07:00:07.609013
Config  : ./aria.localhost/private/backups/20250117_070003-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250117_070003-aria_localhost-database.sql.gz         605.8KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250117_173005-abc_localhost-database.sql.gz

Backup Summary for abc.localhost at 2025-01-17 17:30:08.347196
Config  : ./abc.localhost/private/backups/20250117_173005-abc_localhost-site_config_backup.json 94.0B
Database: ./abc.localhost/private/backups/20250117_173005-abc_localhost-database.sql.gz         272.9KiB
Backup for Site abc.localhost has been successfully completed
File ./aria.localhost/private/backups/20250116_190003-aria_localhost-site_config_backup.json is recent
File ./aria.localhost/private/backups/20250116_190003-aria_localhost-database.sql.gz is recent
File ./aria.localhost/private/backups/20250116_070003-aria_localhost-site_config_backup.json is older than 24 hours
File ./aria.localhost/private/backups/20250117_010004-aria_localhost-database.sql.gz is recent
File ./aria.localhost/private/backups/20250116_130004-aria_localhost-site_config_backup.json is recent
File ./aria.localhost/private/backups/20250117_070003-aria_localhost-site_config_backup.json is recent
File ./aria.localhost/private/backups/20250117_070003-aria_localhost-database.sql.gz is recent
File ./aria.localhost/private/backups/20250117_010004-aria_localhost-site_config_backup.json is recent
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250117_130008-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-17 13:00:12.362049
Config  : ./aria.localhost/private/backups/20250117_130008-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250117_130008-aria_localhost-database.sql.gz         611.8KiB
Backup for Site aria.localhost has been successfully completed
set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz

b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
Backup failed for Site abc.localhost. Database or site_config.json may be corrupted
Traceback with variables (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 848, in backup
    odb = scheduled_backup(
      context = {'sites': ['abc.localhost', 'aria.localhost'], 'force': False, 'verbose': True, 'profile': False}
      with_files = False
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_backup_conf = False
      verbose = True
      compress = False
      include = ''
      exclude = ''
      old_backup_metadata = False
      scheduled_backup = <function scheduled_backup at 0xffffbbdf3880>
      exit_code = 0
      rollback_callback = None
      site = 'abc.localhost'
  File "apps/frappe/frappe/utils/backups.py", line 568, in scheduled_backup
    return new_backup(
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffffbb3c37f0>
  File "apps/frappe/frappe/utils/backups.py", line 626, in new_backup
    odb.get_backup(older_than, ignore_files, force=force)
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffffbb3c37f0>
      odb = <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3880>
  File "apps/frappe/frappe/utils/backups.py", line 195, in get_backup
    self.delete_if_step_fails(self.take_dump, self.backup_path_db)
      self = <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3880>
      older_than = 6
      ignore_files = True
      force = True
      last_db = False
      last_file = False
      last_private_file = False
      site_config_backup_path = False
  File "apps/frappe/frappe/utils/backups.py", line 520, in delete_if_step_fails
    raise e
      self = <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3880>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3880>>
      paths = ('./abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz',)
      path = './abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 515, in delete_if_step_fails
    step()
      self = <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3880>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3880>>
      paths = ('./abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz',)
      path = './abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 468, in take_dump
    frappe.utils.execute_in_shell(command, low_priority=True, check_exit_code=True)
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      frappe = <module 'frappe' from 'apps/frappe/frappe/__init__.py'>
      get_app_branch = <function get_app_branch at 0xffffbbdd77f0>
      gzip_exc = '/usr/bin/gzip'
      database_header_content = ['begin frappe metadata', '[frappe]', 'version = 15.50.0', 'branch = version-15', 'end frappe metadata', '']
      generated_header = '-- begin frappe metadata\n-- [frappe]\n-- version = 15.50.0\n-- branch = version-15\n-- end frappe metadata\n-- \n'
      f = <_io.TextIOWrapper name='./abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz' encoding='UTF-8'>
      cmd = ['/usr/bin/mariadb-dump', '--user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399']
      extra = []
      get_command = <function get_command at 0xffffbbdcf010>
      bin = '/usr/bin/mariadb-dump'
      args = ['--user=_32a6c6f08bcb8399', '--host=127.0.0.1', '--port=3306', '--password=Gmg46SF4xwnWqGxJ', '--single-transaction', '--quick', '--lock-tables=false', '_32a6c6f08bcb8399']
      bin_name = 'mariadb-dump'
      command = 'set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz'
      self = <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3880>
  File "apps/frappe/frappe/utils/__init__.py", line 484, in execute_in_shell
    raise frappe.CommandFailedError(
      cmd = 'set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz'
      verbose = False
      low_priority = True
      check_exit_code = True
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      tempfile = <module 'tempfile' from '/usr/lib/python3.10/tempfile.py'>
      Popen = <class 'subprocess.Popen'>
      stdout = <_io.BufferedRandom name=6>
      stderr = <_io.BufferedRandom name=7>
      kwargs = <exception while printing> Traceback (most recent call last):
          File "env/lib/python3.10/site-packages/traceback_with_variables/core.py", line 222, in _to_cropped_str
            raw = print_(obj)
          File "apps/frappe/frappe/utils/__init__.py", line 326, in dict_printer
            v = deepcopy(v)
          File "/usr/lib/python3.10/copy.py", line 146, in deepcopy
            y = copier(x, memo)
          File "/usr/lib/python3.10/copy.py", line 231, in _deepcopy_dict
            y[deepcopy(key, memo)] = deepcopy(value, memo)
          File "/usr/lib/python3.10/copy.py", line 161, in deepcopy
            rv = reductor(4)
        TypeError: cannot pickle '_io.BufferedRandom' object
        
      p = <Popen: returncode: 2 args: 'set -o pipefail; /usr/bin/mariadb-dump --user=_...>
      exit_code = 2
      out = b''
      err = b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
      failed = 2
frappe.exceptions.CommandFailedError: Command failed
File ./abc.localhost/private/backups/20250117_173005-abc_localhost-database.sql.gz is recent
File ./abc.localhost/private/backups/20250117_173005-abc_localhost-site_config_backup.json is recent
set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz

b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
Backup failed for Site aria.localhost. Database or site_config.json may be corrupted
Traceback with variables (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 848, in backup
    odb = scheduled_backup(
      context = {'sites': ['abc.localhost', 'aria.localhost'], 'force': False, 'verbose': True, 'profile': False}
      with_files = False
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_backup_conf = False
      verbose = True
      compress = False
      include = ''
      exclude = ''
      old_backup_metadata = False
      scheduled_backup = <function scheduled_backup at 0xffffbbdf3880>
      exit_code = 1
      rollback_callback = None
      site = 'aria.localhost'
  File "apps/frappe/frappe/utils/backups.py", line 568, in scheduled_backup
    return new_backup(
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffffbb3c2200>
  File "apps/frappe/frappe/utils/backups.py", line 626, in new_backup
    odb.get_backup(older_than, ignore_files, force=force)
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffffbb3c2200>
      odb = <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3760>
  File "apps/frappe/frappe/utils/backups.py", line 195, in get_backup
    self.delete_if_step_fails(self.take_dump, self.backup_path_db)
      self = <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3760>
      older_than = 6
      ignore_files = True
      force = True
      last_db = False
      last_file = False
      last_private_file = False
      site_config_backup_path = False
  File "apps/frappe/frappe/utils/backups.py", line 520, in delete_if_step_fails
    raise e
      self = <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3760>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3760>>
      paths = ('./abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz',)
      path = './abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 515, in delete_if_step_fails
    step()
      self = <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3760>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3760>>
      paths = ('./abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz',)
      path = './abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 468, in take_dump
    frappe.utils.execute_in_shell(command, low_priority=True, check_exit_code=True)
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      frappe = <module 'frappe' from 'apps/frappe/frappe/__init__.py'>
      get_app_branch = <function get_app_branch at 0xffffbbdd77f0>
      gzip_exc = '/usr/bin/gzip'
      database_header_content = ['begin frappe metadata', '[frappe]', 'version = 15.50.0', 'branch = version-15', 'end frappe metadata', '']
      generated_header = '-- begin frappe metadata\n-- [frappe]\n-- version = 15.50.0\n-- branch = version-15\n-- end frappe metadata\n-- \n'
      f = <_io.TextIOWrapper name='./abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz' encoding='UTF-8'>
      cmd = ['/usr/bin/mariadb-dump', '--user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399']
      extra = []
      get_command = <function get_command at 0xffffbbdcf010>
      bin = '/usr/bin/mariadb-dump'
      args = ['--user=_32a6c6f08bcb8399', '--host=127.0.0.1', '--port=3306', '--password=Gmg46SF4xwnWqGxJ', '--single-transaction', '--quick', '--lock-tables=false', '_32a6c6f08bcb8399']
      bin_name = 'mariadb-dump'
      command = 'set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz'
      self = <frappe.utils.backups.BackupGenerator object at 0xffffbb3c3760>
  File "apps/frappe/frappe/utils/__init__.py", line 484, in execute_in_shell
    raise frappe.CommandFailedError(
      cmd = 'set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250117_233009-abc_localhost-database.sql.gz'
      verbose = False
      low_priority = True
      check_exit_code = True
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      tempfile = <module 'tempfile' from '/usr/lib/python3.10/tempfile.py'>
      Popen = <class 'subprocess.Popen'>
      stdout = <_io.BufferedRandom name=6>
      stderr = <_io.BufferedRandom name=7>
      kwargs = <exception while printing> Traceback (most recent call last):
          File "env/lib/python3.10/site-packages/traceback_with_variables/core.py", line 222, in _to_cropped_str
            raw = print_(obj)
          File "apps/frappe/frappe/utils/__init__.py", line 326, in dict_printer
            v = deepcopy(v)
          File "/usr/lib/python3.10/copy.py", line 146, in deepcopy
            y = copier(x, memo)
          File "/usr/lib/python3.10/copy.py", line 231, in _deepcopy_dict
            y[deepcopy(key, memo)] = deepcopy(value, memo)
          File "/usr/lib/python3.10/copy.py", line 161, in deepcopy
            rv = reductor(4)
        TypeError: cannot pickle '_io.BufferedRandom' object
        
      p = <Popen: returncode: 2 args: 'set -o pipefail; /usr/bin/mariadb-dump --user=_...>
      exit_code = 2
      out = b''
      err = b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
      failed = 2
frappe.exceptions.CommandFailedError: Command failed
set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz

b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
Backup failed for Site abc.localhost. Database or site_config.json may be corrupted
Traceback with variables (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 848, in backup
    odb = scheduled_backup(
      context = {'sites': ['abc.localhost', 'aria.localhost'], 'force': False, 'verbose': True, 'profile': False}
      with_files = False
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_backup_conf = False
      verbose = True
      compress = False
      include = ''
      exclude = ''
      old_backup_metadata = False
      scheduled_backup = <function scheduled_backup at 0xffff8a3c3880>
      exit_code = 0
      rollback_callback = None
      site = 'abc.localhost'
  File "apps/frappe/frappe/utils/backups.py", line 568, in scheduled_backup
    return new_backup(
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffff899937f0>
  File "apps/frappe/frappe/utils/backups.py", line 626, in new_backup
    odb.get_backup(older_than, ignore_files, force=force)
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffff899937f0>
      odb = <frappe.utils.backups.BackupGenerator object at 0xffff89993880>
  File "apps/frappe/frappe/utils/backups.py", line 195, in get_backup
    self.delete_if_step_fails(self.take_dump, self.backup_path_db)
      self = <frappe.utils.backups.BackupGenerator object at 0xffff89993880>
      older_than = 6
      ignore_files = True
      force = True
      last_db = False
      last_file = False
      last_private_file = False
      site_config_backup_path = False
  File "apps/frappe/frappe/utils/backups.py", line 520, in delete_if_step_fails
    raise e
      self = <frappe.utils.backups.BackupGenerator object at 0xffff89993880>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffff89993880>>
      paths = ('./abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz',)
      path = './abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 515, in delete_if_step_fails
    step()
      self = <frappe.utils.backups.BackupGenerator object at 0xffff89993880>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffff89993880>>
      paths = ('./abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz',)
      path = './abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 468, in take_dump
    frappe.utils.execute_in_shell(command, low_priority=True, check_exit_code=True)
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      frappe = <module 'frappe' from 'apps/frappe/frappe/__init__.py'>
      get_app_branch = <function get_app_branch at 0xffff8a3a77f0>
      gzip_exc = '/usr/bin/gzip'
      database_header_content = ['begin frappe metadata', '[frappe]', 'version = 15.50.0', 'branch = version-15', 'end frappe metadata', '']
      generated_header = '-- begin frappe metadata\n-- [frappe]\n-- version = 15.50.0\n-- branch = version-15\n-- end frappe metadata\n-- \n'
      f = <_io.TextIOWrapper name='./abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz' encoding='UTF-8'>
      cmd = ['/usr/bin/mariadb-dump', '--user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399']
      extra = []
      get_command = <function get_command at 0xffff8a39f010>
      bin = '/usr/bin/mariadb-dump'
      args = ['--user=_32a6c6f08bcb8399', '--host=127.0.0.1', '--port=3306', '--password=Gmg46SF4xwnWqGxJ', '--single-transaction', '--quick', '--lock-tables=false', '_32a6c6f08bcb8399']
      bin_name = 'mariadb-dump'
      command = 'set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz'
      self = <frappe.utils.backups.BackupGenerator object at 0xffff89993880>
  File "apps/frappe/frappe/utils/__init__.py", line 484, in execute_in_shell
    raise frappe.CommandFailedError(
      cmd = 'set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz'
      verbose = False
      low_priority = True
      check_exit_code = True
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      tempfile = <module 'tempfile' from '/usr/lib/python3.10/tempfile.py'>
      Popen = <class 'subprocess.Popen'>
      stdout = <_io.BufferedRandom name=6>
      stderr = <_io.BufferedRandom name=7>
      kwargs = <exception while printing> Traceback (most recent call last):
          File "env/lib/python3.10/site-packages/traceback_with_variables/core.py", line 222, in _to_cropped_str
            raw = print_(obj)
          File "apps/frappe/frappe/utils/__init__.py", line 326, in dict_printer
            v = deepcopy(v)
          File "/usr/lib/python3.10/copy.py", line 146, in deepcopy
            y = copier(x, memo)
          File "/usr/lib/python3.10/copy.py", line 231, in _deepcopy_dict
            y[deepcopy(key, memo)] = deepcopy(value, memo)
          File "/usr/lib/python3.10/copy.py", line 161, in deepcopy
            rv = reductor(4)
        TypeError: cannot pickle '_io.BufferedRandom' object
        
      p = <Popen: returncode: 2 args: 'set -o pipefail; /usr/bin/mariadb-dump --user=_...>
      exit_code = 2
      out = b''
      err = b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
      failed = 2
frappe.exceptions.CommandFailedError: Command failed
File ./abc.localhost/private/backups/20250117_173005-abc_localhost-database.sql.gz is recent
File ./abc.localhost/private/backups/20250117_173005-abc_localhost-site_config_backup.json is recent
set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz

b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
Backup failed for Site aria.localhost. Database or site_config.json may be corrupted
Traceback with variables (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 848, in backup
    odb = scheduled_backup(
      context = {'sites': ['abc.localhost', 'aria.localhost'], 'force': False, 'verbose': True, 'profile': False}
      with_files = False
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_backup_conf = False
      verbose = True
      compress = False
      include = ''
      exclude = ''
      old_backup_metadata = False
      scheduled_backup = <function scheduled_backup at 0xffff8a3c3880>
      exit_code = 1
      rollback_callback = None
      site = 'aria.localhost'
  File "apps/frappe/frappe/utils/backups.py", line 568, in scheduled_backup
    return new_backup(
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffff89992200>
  File "apps/frappe/frappe/utils/backups.py", line 626, in new_backup
    odb.get_backup(older_than, ignore_files, force=force)
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffff89992200>
      odb = <frappe.utils.backups.BackupGenerator object at 0xffff89993760>
  File "apps/frappe/frappe/utils/backups.py", line 195, in get_backup
    self.delete_if_step_fails(self.take_dump, self.backup_path_db)
      self = <frappe.utils.backups.BackupGenerator object at 0xffff89993760>
      older_than = 6
      ignore_files = True
      force = True
      last_db = False
      last_file = False
      last_private_file = False
      site_config_backup_path = False
  File "apps/frappe/frappe/utils/backups.py", line 520, in delete_if_step_fails
    raise e
      self = <frappe.utils.backups.BackupGenerator object at 0xffff89993760>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffff89993760>>
      paths = ('./abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz',)
      path = './abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 515, in delete_if_step_fails
    step()
      self = <frappe.utils.backups.BackupGenerator object at 0xffff89993760>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffff89993760>>
      paths = ('./abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz',)
      path = './abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 468, in take_dump
    frappe.utils.execute_in_shell(command, low_priority=True, check_exit_code=True)
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      frappe = <module 'frappe' from 'apps/frappe/frappe/__init__.py'>
      get_app_branch = <function get_app_branch at 0xffff8a3a77f0>
      gzip_exc = '/usr/bin/gzip'
      database_header_content = ['begin frappe metadata', '[frappe]', 'version = 15.50.0', 'branch = version-15', 'end frappe metadata', '']
      generated_header = '-- begin frappe metadata\n-- [frappe]\n-- version = 15.50.0\n-- branch = version-15\n-- end frappe metadata\n-- \n'
      f = <_io.TextIOWrapper name='./abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz' encoding='UTF-8'>
      cmd = ['/usr/bin/mariadb-dump', '--user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399']
      extra = []
      get_command = <function get_command at 0xffff8a39f010>
      bin = '/usr/bin/mariadb-dump'
      args = ['--user=_32a6c6f08bcb8399', '--host=127.0.0.1', '--port=3306', '--password=Gmg46SF4xwnWqGxJ', '--single-transaction', '--quick', '--lock-tables=false', '_32a6c6f08bcb8399']
      bin_name = 'mariadb-dump'
      command = 'set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz'
      self = <frappe.utils.backups.BackupGenerator object at 0xffff89993760>
  File "apps/frappe/frappe/utils/__init__.py", line 484, in execute_in_shell
    raise frappe.CommandFailedError(
      cmd = 'set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250118_053004-abc_localhost-database.sql.gz'
      verbose = False
      low_priority = True
      check_exit_code = True
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      tempfile = <module 'tempfile' from '/usr/lib/python3.10/tempfile.py'>
      Popen = <class 'subprocess.Popen'>
      stdout = <_io.BufferedRandom name=6>
      stderr = <_io.BufferedRandom name=7>
      kwargs = <exception while printing> Traceback (most recent call last):
          File "env/lib/python3.10/site-packages/traceback_with_variables/core.py", line 222, in _to_cropped_str
            raw = print_(obj)
          File "apps/frappe/frappe/utils/__init__.py", line 326, in dict_printer
            v = deepcopy(v)
          File "/usr/lib/python3.10/copy.py", line 146, in deepcopy
            y = copier(x, memo)
          File "/usr/lib/python3.10/copy.py", line 231, in _deepcopy_dict
            y[deepcopy(key, memo)] = deepcopy(value, memo)
          File "/usr/lib/python3.10/copy.py", line 161, in deepcopy
            rv = reductor(4)
        TypeError: cannot pickle '_io.BufferedRandom' object
        
      p = <Popen: returncode: 2 args: 'set -o pipefail; /usr/bin/mariadb-dump --user=_...>
      exit_code = 2
      out = b''
      err = b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
      failed = 2
frappe.exceptions.CommandFailedError: Command failed
set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz

b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
Backup failed for Site abc.localhost. Database or site_config.json may be corrupted
Traceback with variables (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 848, in backup
    odb = scheduled_backup(
      context = {'sites': ['abc.localhost', 'aria.localhost'], 'force': False, 'verbose': True, 'profile': False}
      with_files = False
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_backup_conf = False
      verbose = True
      compress = False
      include = ''
      exclude = ''
      old_backup_metadata = False
      scheduled_backup = <function scheduled_backup at 0xffffb2ee3880>
      exit_code = 0
      rollback_callback = None
      site = 'abc.localhost'
  File "apps/frappe/frappe/utils/backups.py", line 568, in scheduled_backup
    return new_backup(
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffffb24b37f0>
  File "apps/frappe/frappe/utils/backups.py", line 626, in new_backup
    odb.get_backup(older_than, ignore_files, force=force)
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffffb24b37f0>
      odb = <frappe.utils.backups.BackupGenerator object at 0xffffb24b3880>
  File "apps/frappe/frappe/utils/backups.py", line 195, in get_backup
    self.delete_if_step_fails(self.take_dump, self.backup_path_db)
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb24b3880>
      older_than = 6
      ignore_files = True
      force = True
      last_db = False
      last_file = False
      last_private_file = False
      site_config_backup_path = False
  File "apps/frappe/frappe/utils/backups.py", line 520, in delete_if_step_fails
    raise e
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb24b3880>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffffb24b3880>>
      paths = ('./abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz',)
      path = './abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 515, in delete_if_step_fails
    step()
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb24b3880>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffffb24b3880>>
      paths = ('./abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz',)
      path = './abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 468, in take_dump
    frappe.utils.execute_in_shell(command, low_priority=True, check_exit_code=True)
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      frappe = <module 'frappe' from 'apps/frappe/frappe/__init__.py'>
      get_app_branch = <function get_app_branch at 0xffffb2ec77f0>
      gzip_exc = '/usr/bin/gzip'
      database_header_content = ['begin frappe metadata', '[frappe]', 'version = 15.50.0', 'branch = version-15', 'end frappe metadata', '']
      generated_header = '-- begin frappe metadata\n-- [frappe]\n-- version = 15.50.0\n-- branch = version-15\n-- end frappe metadata\n-- \n'
      f = <_io.TextIOWrapper name='./abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz' encoding='UTF-8'>
      cmd = ['/usr/bin/mariadb-dump', '--user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399']
      extra = []
      get_command = <function get_command at 0xffffb2ebf010>
      bin = '/usr/bin/mariadb-dump'
      args = ['--user=_32a6c6f08bcb8399', '--host=127.0.0.1', '--port=3306', '--password=Gmg46SF4xwnWqGxJ', '--single-transaction', '--quick', '--lock-tables=false', '_32a6c6f08bcb8399']
      bin_name = 'mariadb-dump'
      command = 'set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz'
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb24b3880>
  File "apps/frappe/frappe/utils/__init__.py", line 484, in execute_in_shell
    raise frappe.CommandFailedError(
      cmd = 'set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz'
      verbose = False
      low_priority = True
      check_exit_code = True
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      tempfile = <module 'tempfile' from '/usr/lib/python3.10/tempfile.py'>
      Popen = <class 'subprocess.Popen'>
      stdout = <_io.BufferedRandom name=6>
      stderr = <_io.BufferedRandom name=7>
      kwargs = <exception while printing> Traceback (most recent call last):
          File "env/lib/python3.10/site-packages/traceback_with_variables/core.py", line 222, in _to_cropped_str
            raw = print_(obj)
          File "apps/frappe/frappe/utils/__init__.py", line 326, in dict_printer
            v = deepcopy(v)
          File "/usr/lib/python3.10/copy.py", line 146, in deepcopy
            y = copier(x, memo)
          File "/usr/lib/python3.10/copy.py", line 231, in _deepcopy_dict
            y[deepcopy(key, memo)] = deepcopy(value, memo)
          File "/usr/lib/python3.10/copy.py", line 161, in deepcopy
            rv = reductor(4)
        TypeError: cannot pickle '_io.BufferedRandom' object
        
      p = <Popen: returncode: 2 args: 'set -o pipefail; /usr/bin/mariadb-dump --user=_...>
      exit_code = 2
      out = b''
      err = b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
      failed = 2
frappe.exceptions.CommandFailedError: Command failed
File ./abc.localhost/private/backups/20250117_173005-abc_localhost-database.sql.gz is recent
File ./abc.localhost/private/backups/20250117_173005-abc_localhost-site_config_backup.json is recent
set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz

b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
Backup failed for Site aria.localhost. Database or site_config.json may be corrupted
Traceback with variables (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 848, in backup
    odb = scheduled_backup(
      context = {'sites': ['abc.localhost', 'aria.localhost'], 'force': False, 'verbose': True, 'profile': False}
      with_files = False
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_backup_conf = False
      verbose = True
      compress = False
      include = ''
      exclude = ''
      old_backup_metadata = False
      scheduled_backup = <function scheduled_backup at 0xffffb2ee3880>
      exit_code = 1
      rollback_callback = None
      site = 'aria.localhost'
  File "apps/frappe/frappe/utils/backups.py", line 568, in scheduled_backup
    return new_backup(
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffffb24b2200>
  File "apps/frappe/frappe/utils/backups.py", line 626, in new_backup
    odb.get_backup(older_than, ignore_files, force=force)
      older_than = 6
      ignore_files = True
      backup_path = None
      backup_path_db = None
      backup_path_files = None
      backup_path_private_files = None
      backup_path_conf = None
      ignore_conf = False
      include_doctypes = ''
      exclude_doctypes = ''
      compress = False
      force = True
      verbose = True
      old_backup_metadata = False
      rollback_callback = <frappe.utils.CallbackManager object at 0xffffb24b2200>
      odb = <frappe.utils.backups.BackupGenerator object at 0xffffb24b3760>
  File "apps/frappe/frappe/utils/backups.py", line 195, in get_backup
    self.delete_if_step_fails(self.take_dump, self.backup_path_db)
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb24b3760>
      older_than = 6
      ignore_files = True
      force = True
      last_db = False
      last_file = False
      last_private_file = False
      site_config_backup_path = False
  File "apps/frappe/frappe/utils/backups.py", line 520, in delete_if_step_fails
    raise e
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb24b3760>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffffb24b3760>>
      paths = ('./abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz',)
      path = './abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 515, in delete_if_step_fails
    step()
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb24b3760>
      step = <bound method BackupGenerator.take_dump of <frappe.utils.backups.BackupGenerator object at 0xffffb24b3760>>
      paths = ('./abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz',)
      path = './abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz'
  File "apps/frappe/frappe/utils/backups.py", line 468, in take_dump
    frappe.utils.execute_in_shell(command, low_priority=True, check_exit_code=True)
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      frappe = <module 'frappe' from 'apps/frappe/frappe/__init__.py'>
      get_app_branch = <function get_app_branch at 0xffffb2ec77f0>
      gzip_exc = '/usr/bin/gzip'
      database_header_content = ['begin frappe metadata', '[frappe]', 'version = 15.50.0', 'branch = version-15', 'end frappe metadata', '']
      generated_header = '-- begin frappe metadata\n-- [frappe]\n-- version = 15.50.0\n-- branch = version-15\n-- end frappe metadata\n-- \n'
      f = <_io.TextIOWrapper name='./abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz' encoding='UTF-8'>
      cmd = ['/usr/bin/mariadb-dump', '--user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399']
      extra = []
      get_command = <function get_command at 0xffffb2ebf010>
      bin = '/usr/bin/mariadb-dump'
      args = ['--user=_32a6c6f08bcb8399', '--host=127.0.0.1', '--port=3306', '--password=Gmg46SF4xwnWqGxJ', '--single-transaction', '--quick', '--lock-tables=false', '_32a6c6f08bcb8399']
      bin_name = 'mariadb-dump'
      command = 'set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz'
      self = <frappe.utils.backups.BackupGenerator object at 0xffffb24b3760>
  File "apps/frappe/frappe/utils/__init__.py", line 484, in execute_in_shell
    raise frappe.CommandFailedError(
      cmd = 'set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=Gmg46SF4xwnWqGxJ --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250118_113003-abc_localhost-database.sql.gz'
      verbose = False
      low_priority = True
      check_exit_code = True
      shlex = <module 'shlex' from '/usr/lib/python3.10/shlex.py'>
      tempfile = <module 'tempfile' from '/usr/lib/python3.10/tempfile.py'>
      Popen = <class 'subprocess.Popen'>
      stdout = <_io.BufferedRandom name=6>
      stderr = <_io.BufferedRandom name=7>
      kwargs = <exception while printing> Traceback (most recent call last):
          File "env/lib/python3.10/site-packages/traceback_with_variables/core.py", line 222, in _to_cropped_str
            raw = print_(obj)
          File "apps/frappe/frappe/utils/__init__.py", line 326, in dict_printer
            v = deepcopy(v)
          File "/usr/lib/python3.10/copy.py", line 146, in deepcopy
            y = copier(x, memo)
          File "/usr/lib/python3.10/copy.py", line 231, in _deepcopy_dict
            y[deepcopy(key, memo)] = deepcopy(value, memo)
          File "/usr/lib/python3.10/copy.py", line 161, in deepcopy
            rv = reductor(4)
        TypeError: cannot pickle '_io.BufferedRandom' object
        
      p = <Popen: returncode: 2 args: 'set -o pipefail; /usr/bin/mariadb-dump --user=_...>
      exit_code = 2
      out = b''
      err = b'mariadb-dump: Got error: 2002: "Can\'t connect to server on \'127.0.0.1\' (115)" when trying to connect\n'
      failed = 2
frappe.exceptions.CommandFailedError: Command failed
set -o pipefail; /usr/bin/mariadb-dump --user=_32a6c6f08bcb8399 --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _32a6c6f08bcb8399 | /usr/bin/gzip >> ./abc.localhost/private/backups/20250118_173004-abc_localhost-database.sql.gz

Backup Summary for abc.localhost at 2025-01-18 17:30:08.200216
Config  : ./abc.localhost/private/backups/20250118_173004-abc_localhost-site_config_backup.json 94.0B
Database: ./abc.localhost/private/backups/20250118_173004-abc_localhost-database.sql.gz         272.9KiB
Backup for Site abc.localhost has been successfully completed
File ./aria.localhost/private/backups/20250116_190003-aria_localhost-site_config_backup.json is older than 24 hours
File ./aria.localhost/private/backups/20250117_010004-aria_localhost-database.sql.gz is older than 24 hours
File ./aria.localhost/private/backups/20250116_130004-aria_localhost-site_config_backup.json is older than 24 hours
File ./aria.localhost/private/backups/20250117_070003-aria_localhost-site_config_backup.json is older than 24 hours
File ./aria.localhost/private/backups/20250117_070003-aria_localhost-database.sql.gz is older than 24 hours
File ./aria.localhost/private/backups/20250117_010004-aria_localhost-site_config_backup.json is older than 24 hours
File ./aria.localhost/private/backups/20250117_130008-aria_localhost-site_config_backup.json is recent
File ./aria.localhost/private/backups/20250117_130008-aria_localhost-database.sql.gz is recent
set -o pipefail; /usr/bin/mariadb-dump --user=_8fa9d71ae1ff9bee --host=127.0.0.1 --port=3306 --password=********** --single-transaction --quick --lock-tables=false _8fa9d71ae1ff9bee | /usr/bin/gzip >> ./aria.localhost/private/backups/20250118_130008-aria_localhost-database.sql.gz

Backup Summary for aria.localhost at 2025-01-18 13:00:12.265457
Config  : ./aria.localhost/private/backups/20250118_130008-aria_localhost-site_config_backup.json 94.0B
Database: ./aria.localhost/private/backups/20250118_130008-aria_localhost-database.sql.gz         622.4KiB
Backup for Site aria.localhost has been successfully completed
